### PR TITLE
Hide the Publish date of pinned Categories

### PR DESCRIPTION
**Pull request by belljohan3**

### CODE REVIEW

File: app/src/components/tags/HorizontalScrollableTagViewer.vue [hunk 1]
```diff
@@ -119,6 +122,7 @@ useResizeObserver(scrollContent, setSpinBtnVisibility);
                         v-for="content in taggedDocs"
                         :key="content._id"
                         :content="content"
+                        :show-publish-date="props.showPublishDate"
```
Comment (laurensgroeneveld) [inline app/src/components/tags/HorizontalScrollableTagViewer.vue:125]: `props` is not necessary here, as all props are available by their name in the Vue template.

```suggestion
                        :show-publish-date="showPublishDate"
```

File: app/src/components/tags/HorizontalScrollableTagViewerCollection.vue [hunk 2]
```diff
@@ -7,6 +7,7 @@ type Props = {
     tagType: TagType;
     tagQueryOptions: queryOptions;
     contentQueryOptions: queryOptions;
+    showPublishDate?: boolean;
```
Comment (laurensgroeneveld) [inline app/src/components/tags/HorizontalScrollableTagViewerCollection.vue:10]: I don't think this prop is needed as it's value is never used.

**Review by laurensgroeneveld — COMMENTED** [review]
Just two small comments, but looks good otherwise, nice work on adding a test 👍

**Review by ivanslabbert — APPROVED** [review]
Approved, but please fix this before merging?
https://github.com/bccsa/luminary/pull/249/files#r1689564228

**Pull request merged**